openmdao/core/tests/test_check_derivs.py

```diff
@@ -3552,7 +3598,7 @@ def test_cs_around_newton_top_sparse(self):
         prob.model.linear_solver = om.DirectSolver(assemble_jac=True)
         prob.run_model()
 
-        totals = prob.check_totals(of=['obj', 'con1'], wrt=['x', 'z'], method='cs', out_stream=None)
+        totals = prob.check_totals(of=['obj', 'con1'], wrt=['x', 'z'], method='cs')
```

Comment:
Shouldn't out_stream=None be there for a test?

Reply:
Fixed